apps/storybook/docs/d_applicatief/d_voorbeelden/accordion-popover-menu/accordion-popover-menu.stories-doc.mdx

```diff
@@ -0,0 +1,25 @@
+import { Meta } from '@storybook/addon-docs';
+import { ArgTypes, Canvas } from '@storybook/blocks';
+import * as AccordionPopoverMenuStories from './accordion-popover-menu.stories'
+
+<Meta of={AccordionPopoverMenuStories} />
+
+# Accordion met popover menu's 
+ 
+De accordion is een UI-element dat wordt gebruikt om inhoud te organiseren in een uitklapbaar formaat. Elk accordion-item kan een popover bevatten dat extra opties biedt voor dat specifieke item. 
+ 
+## Gebruik 
+ 
+- De accordion met popover menu's wordt gebruikt om inhoud te structureren en gebruikers de mogelijkheid te geven om specifieke acties uit te voeren op individuele items. 
+ 
+## Ontwerp 
+ 
+- Accordion-item: Elk item bestaat uit een titel en een inhoudsgebied dat kan worden uitgeklapt of ingeklapt. 
+ 
+- Popover menu: Elk accordion-item kan een popover menu bevatten dat extra opties biedt voor dat specifieke item. Het popover menu wordt meestal weergegeven als drie verticale punten (•••). 
+ 
+- Interactie: Gebruikers kunnen op de titel van een accordion-item klikken of tikken om het item uit te klappen of in te klappen. Door op het popover menu te klikken of tikken, wordt een keuzemenu geopend met extra opties voor dat specifieke item. 
+ 
+## Voorbeeld
+
+<Canvas of={AccordionPopoverMenuStories.AccordionPopoverMenu} />
```

Comment:
overweeg om het visueel voorbeeld hier een vaste hoogte mee te geven zodat je zeker 3 levels diep kan openklikken ( maar hoe je dit doet in de nieuwe Storybook 7.x weet ik niet ) - mogelijks moet je een ander element dan `Canvas` gebruiken

Reply:
`export default {
    title: 'Components-next/form/select-next',
    tags: ['autodocs'],
    args: selectArgs,
    argTypes: selectArgTypes,
    parameters: {
        docs: {
            page: selectDocs,
            story: {
                height: '300px',
            },
        },
    },
} as Meta<typeof selectArgs>;`

Reply:
![image](https://github.com/milieuinfo/uigov-web-components/assets/122293433/3e7c43c7-2dfe-4082-b73c-fb093feda4b6)